qemu/tests/qemu_guest_agent.py

```diff
@@ -162,21 +157,18 @@ def gagent_create(self, params, vm, *args):
 
         return self.gagent
 
-    @error.context_aware
     def setup_gagent_in_guest(self, params, vm):
-        error.context("Setup guest agent in VM '%s'" % vm.name)
+        logging.info("Setup guest agent in VM '%s'" % vm.name)
         self.gagent_install(params, vm, *[params.get("gagent_install_cmd")])
         self.gagent_start(params, vm, *[params.get("gagent_start_cmd")])
         args = [params.get("gagent_serial_type"), params.get("gagent_name")]
         self.gagent_create(params, vm, *args)
 
-    @error.context_aware
     def gagent_verify(self, params, vm):
-        error.context("Check if guest agent work.", logging.info)
+        logging.info("Check if guest agent work.",)
 
         if not self.gagent:
-            raise error.TestError("Could not find guest agent object"
-                                  "for VM '%s'" % vm.name)
+            self.test.error("Could not find guest agent object"                                                                                                    "for VM '%s'" % vm.name)
```

Comment:
No )

Reply:
It is problem of display format.  ")" is existing.  I have corrected it.